app/util/data_preparation/api/confluence_clients.py

```diff
@@ -139,6 +139,11 @@ def get_total_pages_count(self):
         response = self.get(api_url, 'Could not get issues count')
         return response.json().get('totalSize', 0)
 
+    def get_collaborative_editing_status(self):
+        api_url = f'{self.host}/rest/synchrony-interop/status'
+        response = self.get(api_url, error_msg='Could not get collaborative editing status').json()
+        return response
+
```

Comment:
Good catch of `status` endpoint! But I figured out that there is option 'Limited' in Collaborative mode. In this case `Only one person will be able to edit at a time.` (doc). 
What do you think about add force enabling of collaborative editing?
`/rest/synchrony-interop/enable` this endpoint helps in this case, it starts a job for enabling CE(1-2sec).

Reply:
Thx that you mentioned about the "Collaborative editing LIMITED"  option. We have to check the "synchronyEnabled" status too! I will add this check obligatorily.
As regards to force enabling of collaborative editing it is a good idea, that's what I thought too! I think I need to spend some time to investigate would it have a bad impact on us/vendors!